packages/synapse-react-client/src/utils/AppUtils/session/ApplicationSessionManager.tsx

```diff
@@ -142,11 +139,7 @@ export function ApplicationSessionManager(
 
   /** Call refreshSession on mount */
   useEffect(() => {
-    if (forceRelogin) {
-      clearSession()
```

Comment:
We just never reached this code because we didn't wait for the app to be initialized before redirecting to One Sage.  Now we are hitting this code in the OAuth sign in app, which causes the page to reload (infinite).  Idea of this PR is to utilize maxAge in order to force re-login.